crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -55,6 +55,31 @@ def fs_for_select(selected)
     )
   end
 
+  def beautify_disk_owner(owner)
+    @owner2label ||= {
+      "Ceph" => t(".disk_role_ceph"),
+      "Cinder" => t(".disk_role_cinder"),
+      "Swift" => t(".disk_role_swift"),
+      "LVM_DRBD" => t(".disk_role_drbd"),
+      "sbd" => t(".disk_role_sbd")
```

Comment:
Don't we need also role for software RAID?
Btw. it is bad that we have different format names for different disk roles. Can we standardize this too?